templates/firefox-ios.xliff

```diff
@@ -620,13 +465,21 @@
         <note>The link that shows more Pocket trending stories</note>
       </trans-unit>
       <trans-unit id="ActivityStream.Pocket.SectionTitle">
-        <source>Recommended by Pocket</source>
+        <source>Trending on Pocket</source>
         <note>Section title label for Recommended by Pocket section</note>
       </trans-unit>
       <trans-unit id="ActivityStream.Pocket.Trending">
         <source>Trending</source>
         <note>The description of a Pocket Story</note>
       </trans-unit>
+      <trans-unit id="ActivityStream.TopSites.RowCount">
+        <source>%d rows</source>
```

Comment:
This fell through the cracks.
Unfortunately, I don't think we support plurals correctly on iOS (see https://bugzilla.mozilla.org/show_bug.cgi?id=1181777). So we will have to find a workaround, like "Rows: %d" instead (we've had to do this in the past)